pkg/services/ngalert/eval/eval.go

```diff
@@ -46,13 +48,27 @@ type expressionService interface {
 }
 
 type conditionEvaluator struct {
+	orgId             int64
 	pipeline          expr.DataPipeline
 	expressionService expressionService
 	condition         models.Condition
 	evalTimeout       time.Duration
+	metrics           *metrics.Eval
 }
 
 func (r *conditionEvaluator) EvaluateRaw(ctx context.Context, now time.Time) (resp *backend.QueryDataResponse, err error) {
+	start := time.Now()
```

Comment:
can we use clock here? We use it in other places, so perhaps it will be consistent.

Reply:
I'm going to hold off on this for now, it looks like the package is no longer maintained.

Reply:
Discussed with Josh, we will use clock for the time being while we figure out long term migration.